### DISCUSSION TITLE
Do you accept PR?

### DISCUSSION THREAD

**Discussion by jcubic** [charmbracelet/vhs#154] [Q&A] [answered]
I would like to contribute because I really want to try this project but it doesn't work on my system. Fedora GNU/Linux. My system version of ttdy is outdated and you can't compile from the source. Probably because of the regular expression that checks the version. I can see if I can modify the code to fix the regex. I don't know Go but I think that I can figure out how to fix regular expression, since I'm familiar with them from different languages.

I've created issue #148 but no one replied.

**Comment by maaslalani** [comment, marked as answer]
Hey, PRs and contributions are always welcome. I'll also take a look at your issue and see if I can fix it. It looks like a bug with the version matching logic.

**Reply by jcubic** [reply]
I think that the regex is ok, it works for git version. I was able to run the tool. The problem is the installation of `ttyd`, it only works when I execute from the build directory but after installation I have an error about missing libwebsockets shared library.

I will try to ask on Unix StackExchange for help.
